cypress-shared/support/adyen/testcase.js

```diff
@@ -0,0 +1,280 @@
+    cy.getIframeBody(selectors.AdyenAdminIframe)
+      .find(selectors.AdyenAdminSectionPB4)
+      .contains('Adyen Webhook Password')
+      .find('input')
+      .should('have.value', vtex.adyenWebhookPassword)
+    cy.getIframeBody(selectors.AdyenAdminIframe)
+      .find(selectors.AdyenAdminSectionPB4)
+      .contains('VTEX App Key')
+      .find('input')
+      .should('have.value', vtex.apiKey)
+    cy.getIframeBody(selectors.AdyenAdminIframe)
+      .find(selectors.AdyenAdminSectionPB6)
+      .contains('VTEX App Token')
+      .find('input')
+      .should('have.value', vtex.apiToken)
+    cy.getIframeBody(selectors.AdyenAdminIframe)
+      .find(selectors.AdyenAdminUsePlatformToggle)
+      .contains('Using Adyen for Platforms')
+      .should('have.text', 'Using Adyen for Platforms')
+  })
+}
+
+export function verifyAdyenPlatformSettings() {
+  it(`Verify adyen platform settings in UI`, updateRetry(2), () => {
+    cy.visit('/admin/app/adyen-for-platforms')
+    cy.contains('Settings').should('be.visible')
+    cy.contains('Settings').should('be.visible').click()
+    cy.get(selectors.AdyenPlatformApiKey).should(
+      'have.value',
+      vtex.adyenPlatformApiKey
+    )
+    cy.get(selectors.AdyenPlatformProductionURI).should(
+      'have.value',
+      vtex.adyenPlatformProductionAPI
+    )
+  })
+}
+
+export function createOnBoardingLink(create) {
+  it('Create the onboarding link', () => {
+    cy.visit('/admin/app/adyen-for-platforms')
+    cy.contains('Adyen for Platforms').should('be.visible')
+    cy.contains('productusqa2').should('be.visible').click()
+    if (create) {
+      cy.intercept('POST', `${vtex.baseUrl}/**`).as('RefreshOnboarding')
+      cy.contains('Create New Link').should('be.visible')
+      cy.contains('Create New Link').click()
+
+      /* eslint-disable jest/valid-expect-in-promise */
+      cy.wait('@RefreshOnboarding')
+        .its('response')
+        .then((response) => {
+          cy.writeFile(accountHolderCodeJson, {
+            accountHolderCode:
+              response.body.data.adyenAccountHolder.accountHolderCode,
+          })
+        })
+
+      cy.get(
+        '.vtex-admin-ui-1o3wdue > .vtex-admin-ui-jdrpky > .vtex-admin-ui-79elbk'
+      ).should('be.visible')
+    } else {
+      cy.get(
+        '.vtex-admin-ui-1o3wdue > .vtex-admin-ui-jdrpky > .vtex-admin-ui-79elbk'
+      ).should('not.exist')
+    }
+  })
+}
+
+export function loginToAdyen() {
+  it('Login to adyen dashboard', updateRetry(2), () => {
+    cy.visit(adyenLoginUrl)
+    cy.get(selectors.AdyenLoginUsername).type(adyenLoginUsername)
+    cy.contains('Next').click()
+    cy.get(selectors.AdyenLoginAccount).type(adyenLoginAccount)
+    cy.get(selectors.AdyenLoginPassword).type(adyenLoginPassword)
+    cy.get(selectors.AdyenLoginSubmit).click()
+  })
+}
+
+export function verifyProductInvoiceTestcase(
+  product,
+  env,
+  externalSeller = false
+) {
+  verifyOrderStatus({
```

Comment:
2.8 transactions will fail when we run this testcase again

So, we need to use regex to match order status
It ca be /handling/